security/imx_hab4/u-boot-spl-sign.csf-template

```diff
@@ -1,5 +1,5 @@
 [Header]
-Version = 4.1
+Version = 4.2
```

Comment:
Why do we need to update the version here?

Reply:
this was recommended to customer from NXP FAE